Ephys tests are failing in github actions

**Describe the bug**
The ephy tests are working locally, but are failing when being run in github actions. There's a deprecration warning though with local tests.

```
======================================================================
ERROR: test_scale_recording (tests.test_compressors.TestEphysCompressors)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/aind-data-transfer/aind-data-transfer/tests/test_compressors.py", line 71, in test_scale_recording
    read_block = next(read_blocks)
  File "/home/runner/work/aind-data-transfer/aind-data-transfer/src/aind_data_transfer/readers.py", line 68, in get_read_blocks
    rec = se.read_openephys(
  File "/opt/hostedtoolcache/Python/3.8.15/x64/lib/python3.8/site-packages/spikeinterface/extractors/neoextractors/openephys.py", line 223, in read_openephys
    recording = OpenEphysBinaryRecordingExtractor(folder_path, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.15/x64/lib/python3.8/site-packages/spikeinterface/extractors/neoextractors/openephys.py", line 136, in __init__
    probe = pi.read_openephys(settings_file=settings_file,
  File "/opt/hostedtoolcache/Python/3.8.15/x64/lib/python3.8/site-packages/probeinterface/io.py", line 1036, in read_openephys
    oe_version = parse(str(info_chain.find("VERSION")))
  File "/opt/hostedtoolcache/Python/3.8.15/x64/lib/python3.8/site-packages/packaging/version.py", line 52, in parse
    return Version(version)
  File "/opt/hostedtoolcache/Python/3.8.15/x64/lib/python3.8/site-packages/packaging/version.py", line 197, in __init__
    raise InvalidVersion(f"Invalid version: '{version}'")
packaging.version.InvalidVersion: Invalid version: '<Element 'VERSION' at 0x7fd1023f0900>'

======================================================================
ERROR: test_read (tests.test_readers.TestEphysReaders)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/aind-data-transfer/aind-data-transfer/tests/test_readers.py", line 50, in test_read
    read_blocks_info = [str(r) for r in read_blocks]
  File "/home/runner/work/aind-data-transfer/aind-data-transfer/tests/test_readers.py", line 50, in <listcomp>
    read_blocks_info = [str(r) for r in read_blocks]
  File "/home/runner/work/aind-data-transfer/aind-data-transfer/src/aind_data_transfer/readers.py", line 68, in get_read_blocks
    rec = se.read_openephys(
  File "/opt/hostedtoolcache/Python/3.8.15/x64/lib/python3.8/site-packages/spikeinterface/extractors/neoextractors/openephys.py", line 223, in read_openephys
    recording = OpenEphysBinaryRecordingExtractor(folder_path, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.15/x64/lib/python3.8/site-packages/spikeinterface/extractors/neoextractors/openephys.py", line 136, in __init__
    probe = pi.read_openephys(settings_file=settings_file,
  File "/opt/hostedtoolcache/Python/3.8.15/x64/lib/python3.8/site-packages/probeinterface/io.py", line 1036, in read_openephys
    oe_version = parse(str(info_chain.find("VERSION")))
  File "/opt/hostedtoolcache/Python/3.8.15/x64/lib/python3.8/site-packages/packaging/version.py", line 52, in parse
    return Version(version)
  File "/opt/hostedtoolcache/Python/3.8.15/x64/lib/python3.8/site-packages/packaging/version.py", line 197, in __init__
    raise InvalidVersion(f"Invalid version: '{version}'")
packaging.version.InvalidVersion: Invalid version: '<Element 'VERSION' at 0x7fd1009a4360>'

----------------------------------------------------------------------
Ran 48 tests in 35.015s

FAILED (errors=2)
Error: Process completed with exit code 1.
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -38,7 +38,7 @@ dev = [
 ]
 ephys = [
     'spikeinterface[full]',
-    'probeinterface>=0.2.11',
+    'probeinterface>=0.2.15',
     'zarr',
     'wavpack-numcodecs>=0.1.3'
 ]

